### ISSUE TITLE
Error converting Pronto IR hex codes to LIRC pulses

### ISSUE THREAD

**Issue by Stichoza** [kiwi-cam/homebridge-broadlink-rm#122]
I'm getting the following error if I use pronto hex code:

```
(node:11099) UnhandledPromiseRejectionWarning: TypeError: log is not a function
    at prontoToLIRC (/usr/lib/node_modules/homebridge-broadlink-rm-pro/helpers/convertProntoCode.js:10:5)
    at convertProntoToBroadlink (/usr/lib/node_modules/homebridge-broadlink-rm-pro/helpers/convertProntoCode.js:68:22)
    at module.exports (/usr/lib/node_modules/homebridge-broadlink-rm-pro/helpers/sendData.js:12:15)
    at TVAccessory.performSend (/usr/lib/node_modules/homebridge-broadlink-rm-pro/accessories/accessory.js:45:7)
    at VolumeSelector.<anonymous> (/usr/lib/node_modules/homebridge-broadlink-rm-pro/accessories/tv.js:374:14)
    at VolumeSelector.emit (events.js:315:20)
    at VolumeSelector.EventEmitter.emit (/usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/EventEmitter.ts:42:22)
    at VolumeSelector.Characteristic._this.setValue (/usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:652:12)
    at /usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Accessory.ts:1439:24
    at Array.forEach (<anonymous>)
(node:11099) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
```

#### Example code: 
```0000 006C 0000 0022 00AD 00AD 0016 0041 0016 0041 0016 0041 0016 0016 0016 0016 0016 0016 0016 0016 0016 0016 0016 0041 0016 0041 0016 0041 0016 0016 0016 0016 0016 0016 0016 0016 0016 0016 0016 0041 0016 0041 0016 0016 0016 0041 0016 0016 0016 0016 0016 0016 0016 0016 0016 0016 0016 0016 0016 0041 0016 0016 0016 0041 0016 0041 0016 0041 0016 0041 0016 06FB```

#### Same code without spaces:
```0000006C0000002200AD00AD0016004100160041001600410016001600160016001600160016001600160016001600410016004100160041001600160016001600160016001600160016001600160041001600410016001600160041001600160016001600160016001600160016001600160016001600410016001600160041001600410016004100160041001606FB```

### Solution:
I used this python code to convert pronto to lirc and it works. - https://gist.github.com/rcarmo/a268972c6173acab8eaa59b56c8d6367

**Comment by kiwi-cam** [comment]
Looks like this error might've been there for a while. This code is untouched from the original fork. Shows how much pronto codes are being used.

I've just pushed homebridge-broadlink-rm-pro@4.3.8-beta.2 which should resolve this error.

**Comment by kiwi-cam** [comment]
This should now be fixed in 4.3.8
